python/invoke_release/tasks.py

```diff
@@ -1111,7 +1172,7 @@ def release(_, verbose=False, no_stash=False):
     version_regular_expression = RE_VERSION
 
     branch_name = _get_branch_name(verbose)
-    if branch_name != BRANCH_MASTER:
+    if branch_name != BRANCH_MASTER and not USE_PULL_REQUEST:
```

Comment:
Dope! Sorry I missed this before. This is actually not correct. When they _start_ this process, the branch should still be master or a release-patching branch. It will only become a pull request branch _after_ this check. So this check should remain unchanged.

Reply:
ah yes, thank for for finding this, I changed this last week when I was just getting started and didn't understand the code yet!